src/print.js

```diff
@@ -0,0 +1,157 @@
+const $todos = document.querySelector('.todoContainer');
+
+export default class Todos {
+  // Define a constructor for the Todos class
+  constructor() {
+    // Retrieve the todoList array from local storage or create an empty array if none exists
+    this.todoList = JSON.parse(localStorage.getItem('todoList')) || [];
+
+    // If the todoList array is not empty, loop through each item and add it to the DOM
+    if (this.todoList.length > 0) {
+      this.todoList.forEach((todo, index) => {
+        // Use insertAdjacentHTML to add a new todo item to the DOM
+        $todos.insertAdjacentHTML(
+          'beforeend',
+          `<ul class="listcontainer ">
+            <div class="checBox">
+              <input type="checkbox" class="checkboxClass" id="myCheckbox">
+            </div>
+            <li class="list textTodo">${todo.description}</li>
+            <i class="fa-solid fa-ellipsis-vertical vertical" ></i>
+            <i class="fa-solid fa-trash-can delete" id="${todo.description}"></i>
+          </ul>
+          `,
+        );
+        // Add a click event listener to the remove button for each todo item
+        const $removeButton = document.getElementById(`${todo.description}`);
+        if ($removeButton) {
+          $removeButton.addEventListener('click', () => {
+            this.remove(todo.index);
+          });
+        }
+        // Update the index property of the todo item
+        todo.index = index;
+        todo.completed = false;
+      });
+    }
```

Comment:
Isn`t it better that the constructor contains only constructors, in your function it is likely to be closed after local storage at line 8?